examples/async_pipeline.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>I would like to see the output and it might be helpful for understanding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-experimental/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>